spring-kafka/src/main/java/org/springframework/kafka/core/KafkaOperations.java

```diff
@@ -267,6 +269,29 @@ default boolean inTransaction() {
 		throw new UnsupportedOperationException("This implementation does not support this operation");
 	}
 
+	/**
+	 * Receive a single record with the default poll timeout (5 seconds).
```

Comment:
I don't think it is legit to mention those `5 seconds` on the interface level. 
Better to say something like "with some default timeout" or leave it as is but without those 5 secs.

(Not so strong opinion: just was confused do not see those 5 seconds in this class)